make/photon/db/registry.sql

```diff
@@ -254,6 +254,19 @@ create table properties (
  UNIQUE (k)
  );
 
+create table label (
+ id int NOT NULL AUTO_INCREMENT,
+ name varchar(128) NOT NULL,
+ description text,
+ color varchar(16),
+ level char(1) NOT NULL,
```

Comment:
better also explain the values of column `scope` and `level` in the `.sql` file?